app/utils/auth.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright 2020 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const { AUTH_MODEL } = require('./auth.consts');
+const AuthClass = require(`./auth.${AUTH_MODEL}`);
```

Comment:
We can't prefix with `./auth`.  This will prevent us from using auth modules installed via NPM later.

Reply:
Allow users to override our auth impl by either pointing to another path or loading from an npm package through $AUTH_MODEL_CLASS variable.